mypyc/codegen/emitclass.py

```diff
@@ -18,6 +19,7 @@
 from mypyc.sametype import is_same_type
 from mypyc.namegen import NameGenerator
 
+PY_3_11 = sys.version_info >= (3, 11)
```

Comment:
I think this isn't right: the running version of Python doesn't necessarily equal the version used at runtime. Maybe we need to emit #ifdef'ed C code instead?

Reply:
Oh right. Yeah my bad.

Reply:
Currently mypyc does not support cross Python version building, but yes conditional C code is better since cross version building is a future goal. The Emitter class does have the `capi_version` attribute for future reference (it's currently always set to the running version of Python).

Reply:
For some reason the ``PY_VERSION_HEX`` checks are not working (I suspect the appropriate headers aren't being included properly). So for now I'm using `capi_version` and leaving `TODO`s to use `PY_VERSION_HEX` in the future.